awx/ui_next/src/screens/InstanceGroup/ContainerGroupDetails/ContainerGroupDetails.jsx

```diff
@@ -1,14 +1,137 @@
+import { Button, Label } from '@patternfly/react-core';
+
+import { VariablesDetail } from '../../../components/CodeMirrorInput';
+import AlertModal from '../../../components/AlertModal';
+import { CardBody, CardActionsRow } from '../../../components/Card';
+import DeleteButton from '../../../components/DeleteButton';
+import {
+  Detail,
+  DetailList,
+  UserDateDetail,
+  DetailBadge,
+} from '../../../components/DetailList';
+import useRequest, { useDismissableError } from '../../../util/useRequest';
+import { jsonToYaml, isJsonString } from '../../../util/yaml';
+import { InstanceGroupsAPI } from '../../../api';
+
+const Unavailable = styled.span`
+  color: var(--pf-global--danger-color--200);
+`;
+
+function ContainerGroupDetails({ instanceGroup, i18n }) {
+  const { id, name } = instanceGroup;
+
+  const history = useHistory();
+
+  const {
+    request: deleteInstanceGroup,
+    isLoading,
+    error: deleteError,
+  } = useRequest(
+    useCallback(async () => {
+      await InstanceGroupsAPI.destroy(id);
+      history.push(`/instance_groups`);
+    }, [id, history])
+  );
+
+  const { error, dismissError } = useDismissableError(deleteError);
 
-function ContainerGroupDetails() {
   return (
-    <PageSection>
-      <Card>
-        <div>Container group details</div>
-      </Card>
-    </PageSection>
+    <CardBody>
+      <DetailList>
+        <Detail
+          label={i18n._(t`Name`)}
+          value={instanceGroup.name}
+          dataCy="container-group-detail-name"
+        />
+        <Detail
+          label={i18n._(t`Type`)}
+          value={i18n._(t`Container group`)}
+          dataCy="container-group-type"
+        />
+        <Detail
+          label={i18n._(t`Credential`)}
+          value={
+            <Label variant="outline" color="blue">
+              {instanceGroup.summary_fields.credential.name}
+            </Label>
+          }
+          dataCy="container-group-credential"
+        />
+        {instanceGroup.capacity ? (
+          <DetailBadge
+            label={i18n._(t`Used capacity`)}
+            content={`${Math.round(
+              100 - instanceGroup.percent_capacity_remaining
+            )} %`}
+            dataCy="container-group-used-capacity"
+          />
+        ) : (
+          <Detail
+            label={i18n._(t`Used capacity`)}
+            value={<Unavailable>{i18n._(t`Unavailable`)}</Unavailable>}
+            dataCy="instance-group-used-capacity"
+          />
+        )}
+        <UserDateDetail
+          label={i18n._(t`Created`)}
+          date={instanceGroup.created}
+          user={instanceGroup.summary_fields.created_by}
+        />
+        <UserDateDetail
+          label={i18n._(t`Last Modified`)}
+          date={instanceGroup.modified}
+          user={instanceGroup.summary_fields.modified_by}
+        />
+        <VariablesDetail
+          label={i18n._(t`Pod spec override`)}
+          value={
+            isJsonString(instanceGroup.pod_spec_override)
```

Comment:
@nixocio To avoid showing a blank configuration, we should use the default value when a `pod_spec_override` hasn't been provided.

- First we make an options request to `api/v2/instance_groups`. 
  - You can include that with https://github.com/ansible/awx/pull/8078/files#diff-beacbbbedd3badd5d65385225d30c291R37 using `Promise.all` if you'd like.
- when `instanceGroup.pod_spec_override` isn't defined, use `actions.PUT.pod_spec_override.default` from the options data instead

Reply:
Thanks for your review @jakemcdermott, I did think about that...but we currently do not do that in the `classic` ui. I am +1 to add that, but I verified the api itself, and it is an empty string when no `pod_spec_override` was provided.  I can update that for sure.

Reply:
Updated. Please, see pictures above.